### PR TITLE
move_BMDC_scripts_to_biomedical_subdirectory

### PR DESCRIPTION
**Pull request by spiekos**
create and move all bmdc scripts to `biomedical` subdirectory

### CODE REVIEW

File: scripts/biomedical/NCBI_Assembly/format_NCBI_Chromosome.py [hunk 1]
```diff
@@ -70,7 +70,9 @@ def write_genome_assembly(file_output, dict_genome_assembly, genome,
         'Taxid':
             'ncbiTaxonID',
         'BioProject':
```
Comment (pradh) [inline scripts/biomedical/NCBI_Assembly/format_NCBI_Chromosome.py:72]: To confirm, are changes in this file intentional?

File: scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Data Commons Scripts
+
+These are scripts used to format data into MCF or TMCF format for ingestion into Data Commons.
```
Comment (pradh) [inline scripts/README.md:3]: TMCF+CSV

**Review by pradh — APPROVED** [review]
Nice!

File: scripts/biomedical/NCBI_Assembly/format_NCBI_Chromosome.py (hunk 1 above)
Comment (spiekos) [reply scripts/biomedical/NCBI_Assembly/format_NCBI_Chromosome.py:85]: yes

File: scripts/README.md (hunk 2 above)
Comment (spiekos) [reply scripts/README.md]: fixed

File: scripts/biomedical/NCBI_Assembly/format_NCBI_Chromosome.py [hunk 3]
```diff
@@ -34,18 +34,31 @@ def format_camel_case(value):
...
     @return    The camel case formatted string
     '''
     value = value.replace(' ', '')
     value_split = re.split('_|-', value)
     value_formatted = ''
     # camel case enum type
     for i in range(len(value_split)):
-        value_formatted = value_formatted + value_split[i][0].upper(
-        ) + value_split[i][1:]
+        value_formatted = (value_formatted +
```
Comment (pradh) [inline scripts/biomedical/NCBI_Assembly/format_NCBI_Chromosome.py:45]:
```suggestion
        value_formatted = value_formatted + value_split[i].capitalize()
                                         
```

File: scripts/biomedical/NCBI_Assembly/format_NCBI_Chromosome.py [hunk 4]
```diff
@@ -34,18 +34,31 @@ def format_camel_case(value):
...
+    
+    @value    The string that needs to be converted to an int
+    @key.     The property name to write the value to
+    @w        Output file to write to
+    '''
+    v = ''
+    for i in value:
+        if i.isdigit():
+            v += i
+    w.write(key + ': ' + v + '\n')
```
Comment (pradh) [inline scripts/biomedical/NCBI_Assembly/format_NCBI_Chromosome.py:62]:
```suggestion

```

File: scripts/biomedical/NCBI_Assembly/format_NCBI_Chromosome.py [hunk 5]
```diff
@@ -34,18 +34,31 @@ def format_camel_case(value):
...
     value = value.replace(' ', '')
     value_split = re.split('_|-', value)
     value_formatted = ''
     # camel case enum type
     for i in range(len(value_split)):
-        value_formatted = value_formatted + value_split[i][0].upper(
-        ) + value_split[i][1:]
+        value_formatted = (value_formatted + 
+        value_split[i][0].upper() + value_split[i][1:])
     return (value_formatted)
```
Comment (pradh) [inline scripts/biomedical/NCBI_Assembly/format_NCBI_Chromosome.py:48]:
```suggestion


```

File: scripts/biomedical/NCBI_Assembly/format_NCBI_Chromosome.py (hunk 3 above)
Comment (pradh) [inline scripts/biomedical/NCBI_Assembly/format_NCBI_Chromosome.py:45]:
```suggestion
        value_formatted = value_formatted + value_split[i].capitalize()
```

File: scripts/biomedical/NCBI_Assembly/format_NCBI_Chromosome.py [hunk 6]
```diff
@@ -34,18 +34,31 @@ def format_camel_case(value):
...
     '''
     value = value.replace(' ', '')
     value_split = re.split('_|-', value)
     value_formatted = ''
     # camel case enum type
     for i in range(len(value_split)):
-        value_formatted = value_formatted + value_split[i][0].upper(
-        ) + value_split[i][1:]
+        value_formatted = (value_formatted + 
+        value_split[i][0].upper() + value_split[i][1:])
```
Comment (pradh) [inline scripts/biomedical/NCBI_Assembly/format_NCBI_Chromosome.py:46]:
```suggestion
```

**Review by pradh — APPROVED** [review]

**Pull request merged**